Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/TensorizationModels.cs

```diff
@@ -0,0 +1,692 @@
+﻿using Microsoft.AI.MachineLearning;
+using Microsoft.AI.MachineLearning.Experimental;
+
+namespace WinMLSamplesGallery.Samples
+{
+    public static class TensorizationModels {
+        enum OnnxDataType : long {
+            UNDEFINED = 0,
+            // Basic types.
+            FLOAT = 1,
+            UINT8 = 2,
+            INT8 = 3,
+            UINT16 = 4,
+            INT16 = 5,
+            INT32 = 6,
+            INT64 = 7,
+            STRING = 8,
+            BOOL = 9,
+
+            // IEEE754 half-precision floating-point format (16 bits wide).
+            // This format has 1 sign bit, 5 exponent bits, and 10 mantissa bits.
+            FLOAT16 = 10,
+
+            DOUBLE = 11,
+            UINT32 = 12,
+            UINT64 = 13,
+            COMPLEX64 = 14,     // complex with float32 real and imaginary components
+            COMPLEX128 = 15,    // complex with float64 real and imaginary components
+
+            // Non-IEEE floating-point format based on IEEE754 single-precision
+            // floating-point number truncated to 16 bits.
+            // This format has 1 sign bit, 8 exponent bits, and 7 mantissa bits.
+            BFLOAT16 = 16,
+        }
+
+#pragma warning disable CA1416 // Validate platform compatibility
+        public static LearningModel IdentityNCHW()

```

Comment:
seems like each of these methods could use a comment saying what they do

Reply:
@smk2007 can better answer this

Reply:
Yes, a comment will help.

Reply:
@smk2007 said he would update these but we shouldn't block on these for now